plugins/schedule/base/lib/api/KalturaScheduleEvent.php

```diff
@@ -352,6 +406,25 @@ public function validateForUpdate($sourceObject, $propertiesToSkip = array())
 			$this->id = $sourceObject->getId();
 		}
 		
+		$linkedTo = $sourceObject->getLinkedTo();
+		if ($this->linkedTo && $this->linkedTo->getEventId() && $this->startDate)

```

Comment:
How can this happen if you already validated it in the insertion?

Reply:
in validateForUpdate, if $this->linkedTo is not null it means that it a property being updated. 
$sourceObject is the event before being updated